app/assets/javascripts/required_fields.coffee

```diff
@@ -373,13 +373,23 @@ $(document).ready ->
   $('.metadata-form, .umm-form').on 'change', 'input[type="radio"], select', ->
     addRequiredIcons(this)
 
+  # MMT-2452: Because the 'clear-radio-button' elements were being handled in draft_form_selectors.coffee,
```

Comment:
I think the explanation in comments is helpful, but I don't think the ticket numbers are (unless it is something that we are going to do or are waiting on)